### DISCUSSION TITLE
The search Feature

### DISCUSSION THREAD

**Discussion by MTelco** [open-metadata/OpenMetadata#13311] [General]
Hi all
we are exploring the search feature in open metadata.
However i see that we can only search my name of dashboards or table only..
while trying to use description, tags as a search, it doesn't work.

Is this a permanent restriction or something to be changed in next releases?

We are setting up this open source tool as our data catalog. if these features are not planned, we might need to look into other tools.?
Regards,
Shyamal

**Comment by hejatibor** [comment]
Hi, 
I had the same problem, then I realized we need to hit the Enter key after writing the pattern into the search field. The dropdown is just an autocomplete hint based on the table names.

Tibor
